dot/sync/syncer_integration_test.go

```diff
@@ -120,7 +120,10 @@ func newTestSyncer(t *testing.T) *Service {
 	cfg.LogLvl = log.Trace
 	mockFinalityGadget := NewMockFinalityGadget(ctrl)
 	mockFinalityGadget.EXPECT().VerifyBlockJustification(gomock.AssignableToTypeOf(common.Hash{}),
-		gomock.AssignableToTypeOf([]byte{})).AnyTimes()
+		gomock.AssignableToTypeOf([]byte{})).DoAndReturn(func(hash common.Hash, justification []byte) ([]byte, error) {
+		return justification, nil
+	}).AnyTimes()
```

Comment:
Is there a way to know exactly how many times this runs?

Reply:
No, since this is in `newTestSyncer` which is used in multiple tests that have different conditions we don't know if this will be called or not.  Ideally the tests that use this would be refactored to put the mocks in-line for each individual test so that we would know exactly how many times it's called.